gatsby/content/projects/clients/gotktrix.mdx

```diff
@@ -0,0 +1,55 @@
+---
+layout: projectimage
+title: gotktrix
+categories:
+ - client
+thumbnail: /docs/projects/images/gotktrix.png
+description: Matrix client in Go and GTK4.
+author: diamondburned
+maturity: alpha
+language: Go
+license: AGPL-3.0-or-later
+repo: https://github.com/diamondburned/gotktrix
+screenshot: /docs/projects/images/gotktrix.png
+slug: gotktrix
+featured: true
+platforms:
+    - Linux
+    - Mac
+features:
+    Room directory: yes
+    Room tag showing: no
+    Room tag editing: no
+    Search joined rooms: yes
+    Room user list: no
+    Display Room Description: no
+    Edit Room Description: no
+    Highlights: no
+    Pushrules: no
+    Send read markers: yes
+    Display read markers: no
+    Sending Invites: no
+    Accepting Invites: no
+    Typing Notification: yes
+    E2E: no
+    Replies: yes
+    Attachment uploading: yes
+    Attachment downloading: no
+    Send stickers: no
+    Send formatted messages (markdown): yes
+    Rich Text Editor for formatted messages: yes
+    Display formatted messages: yes
+    Redacting: no
+    Multiple Matrix Accounts: yes
+    New user registration: no
+    voip: no
+    Reactions: yes
+    Message editing: yes
+    Room upgrades: no
+    Localisations: no
```

Comment:
You need to either put `0` or remove the line, so it stays compatible with the rest of the code base.